packages/availity-react-kit/project/app/Request/components/Patient.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { AvField } from '../../components';
+import { Label, UncontrolledTooltip } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { RequestStore } from '../stores/requestStore';
+
+@observer
+export default class Patient extends Component {
+  constructor(props) {
+    super(props);
+
+    this.updateMemberId = ::this.updateMemberId;
+  }
+
+  updateMemberId(event) {
```

Comment:
Same arrow function here to avoid the constructor boilerplate

Reply:
i'll update